### PR TITLE
The returned value for boolean argument types should be of type string

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
Fixes https://github.com/appium/appium/issues/8321

### CODE REVIEW

File: app/src/main/java/io/appium/uiautomator2/handler/GetElementAttribute.java [hunk 1]
```diff
@@ -38,7 +38,10 @@ public AppiumResponse safeHandle(IHttpRequest request) {
                 return new AppiumResponse(getSessionId(request), WDStatus.SUCCESS, attribute);
             } else {
                 Boolean boolAttribute = element.getBoolAttribute(attributeName);
-                return new AppiumResponse(getSessionId(request), WDStatus.SUCCESS, boolAttribute);
+                // The result should be anyway of type string according to
```
Comment (jlipps) [inline app/src/main/java/io/appium/uiautomator2/handler/GetElementAttribute.java]: NIT: no need for the word `anyway` here?

Comment (mykola-mokhnach) [inline app/src/main/java/io/appium/uiautomator2/handler/GetElementAttribute.java]: natürlich

**Comment by sravanmedarapu** [comment]
Would be great if we also update the corresponding [tests](https://github.com/appium/appium-uiautomator2-server/blob/master/app/src/androidTestE2eTest/java/io/appium/uiautomator2/unittest/test/HandlersTest.java#L773).

**Comment by mykola-mokhnach** [comment]
@sravanmedarapu done

**Pull request merged**